internal/ini/testdata/valid/op_sep_in_values

```diff
@@ -0,0 +1,6 @@
+[default]
+sepInValue = =:[foo]]bar[
+output = json
+[assumerole]
+sepInValue==:[foo]]bar[
+output = json
```

Comment:
The difference in values for `sepInValue` is too subtle to notice :) . Can we add cases with diverse set of values?

Reply:
Could you add more valid/invalid cases like

1.
```
[assumerole]
sepInValue = []
```

2. 
```
[assumerole]
sepInValue = [value] x=a
```
3. 
```
[assumerole]
key[id] = value
```

4.  Want to confirm about nested values additions in future are not impacted in anyway.
```
[assumerole]
   [assumerole2]
        key=value
```
5. 
```
[=:assumerole]
key=value
```
6. 
```
[assumerole] 
key : value
```

7. Nested without spacing
```
[assumerole]
[assumerole2]
key=value
```

For my understanding, most of these are invalid (except 1 and 7 possibly?), is that correct?

Reply:
Case number 5 is not valid FYI as `:`, `=`, `[`, or `]` can't appear in the section statement names.

Reply:
Yeah I was guessing most of those are invalid. 

Yes, case 4 was about service specific nesting support.